src/encord_active/lib/coco/importer.py

```diff
@@ -57,20 +55,18 @@ def upload_img(
         return None
     img_exif = img.getexif()
     if img_exif and (274 in img_exif):  # 274 corresponds to orientation key for EXIF metadata
-        temp_file_name = "temp_image" + file_path.suffix
+        temp_file_name = file_path.name
         img = ImageOps.exif_transpose(img)
-        img.save(temp_file_name)
+        img.save(temp_folder / temp_file_name)
```

Comment:
It's not clear why we need to pass around the temporary directory.
I would probably just use something like
```
import os
from tempfile import TemporaryDirectory
with TemporaryDirectory() as d:
    out_file = os.path.join(d, temp_file_name)
    img.save(out_file)
    try: ...
    except: ...
```
To avoid passing around the directories.

Reply:
The thing is that when we import a coco project and [create a dataset](https://github.com/encord-team/encord-active/blob/ab3eb0b7f6c22c2a29bf8f543e98f4d0d0482a74/src/encord_active/cli/utils/coco.py#L75) for that we just record the paths for each image, then we use those paths in the [create_project](https://github.com/encord-team/encord-active/blob/ab3eb0b7f6c22c2a29bf8f543e98f4d0d0482a74/src/encord_active/cli/utils/coco.py#L77) function, that's why we need to keep images until the end of project creation step. If we remove them just after [this](https://github.com/encord-team/encord-active/blob/ab3eb0b7f6c22c2a29bf8f543e98f4d0d0482a74/src/encord_active/lib/coco/importer.py#L65), they cannot be found later to create the project, if I am not mistaken.